### PR TITLE
Add claims for id, firstName and lastName fields in JWT token

### PR DESCRIPTION
**Pull request by andriimanzar**

### CODE REVIEW

**Comment by knurff** [comment]
|File|Coverage [57.85%]|:green_apple:|
|:-|:-:|:-:|
|[JwtUtils.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FJwtUtils.java)|100%|:green_apple:|
|[RoleParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FRoleParser.java)|100%|:green_apple:|
|[JwtUserDetailsService.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FJwtUserDetailsService.java)|100%|:green_apple:|
|[JwtGenerator.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtGenerator.java)|100%|:green_apple:|
|[JwtTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtTokenParser.java)|100%|:green_apple:|
|[FacebookTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FFacebookTokenParser.java)|89.29%|:green_apple:|
|[JwtUserDetails.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtUserDetails.java)|78%|:green_apple:|
|[FacebookResponseDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookResponseDto.java)|0%|:x:|
|[AuthenticationResponse.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fresponse%2FAuthenticationResponse.java)|0%|:x:|
|[AuthenticationController.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/366eb5b24846240b5f49e399082d9fb9388fd016/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FAuthenticationController.java)|0%|:x:|

|Total Project Coverage|39.06%|:green_apple:|
|:-|:-:|:-:|

**Comment by knurff** [comment]
|File|Coverage [45.69%]|:green_apple:|
|:-|:-:|:-:|
|[JwtUtils.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FJwtUtils.java)|100%|:green_apple:|
|[RoleParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FRoleParser.java)|100%|:green_apple:|
|[JwtUserDetailsService.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FJwtUserDetailsService.java)|100%|:green_apple:|
|[JwtGenerator.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtGenerator.java)|100%|:green_apple:|
|[JwtTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtTokenParser.java)|100%|:green_apple:|
|[FacebookTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FFacebookTokenParser.java)|89.29%|:green_apple:|
|[JwtUserDetails.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtUserDetails.java)|78%|:green_apple:|
|[FacebookResponseDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookResponseDto.java)|0%|:x:|
|[AuthenticationResponse.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fresponse%2FAuthenticationResponse.java)|0%|:x:|
|[AuthenticationController.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/73a020706e6ee1de2c46ce8650d6f0114e7e09e3/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FAuthenticationController.java)|0%|:x:|

|Total Project Coverage|38.14%|:green_apple:|
|:-|:-:|:-:|

**Comment by knurff** [comment]
|File|Coverage [65.94%]|:green_apple:|
|:-|:-:|:-:|
|[JwtUtils.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FJwtUtils.java)|100%|:green_apple:|
|[RoleParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FRoleParser.java)|100%|:green_apple:|
|[JwtUserDetailsService.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FJwtUserDetailsService.java)|100%|:green_apple:|
|[JwtGenerator.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtGenerator.java)|100%|:green_apple:|
|[JwtTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtTokenParser.java)|100%|:green_apple:|
|[FacebookTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FFacebookTokenParser.java)|89.29%|:green_apple:|
|[JwtUserDetails.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtUserDetails.java)|78%|:green_apple:|
|[FacebookTokenDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookTokenDto.java)|0%|:x:|
|[FacebookResponseDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookResponseDto.java)|0%|:x:|
|[AuthenticationResponse.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fresponse%2FAuthenticationResponse.java)|0%|:x:|
|[AuthenticationController.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/beca32fec8f8d29be0d2640888035ecae1b81783/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FAuthenticationController.java)|0%|:x:|

|Total Project Coverage|39.62%|:green_apple:|
|:-|:-:|:-:|

File: src/main/java/com/intellias/intellistart/interviewplanning/security/JwtUserDetails.java [hunk 1]
```diff
@@ -50,4 +53,27 @@ public boolean isEnabled() {
     return true;
   }
 
+  public String getFirstName() {
```
Comment (ram333n) [inline src/main/java/com/intellias/intellistart/interviewplanning/security/JwtUserDetails.java:56]: You can use `@Getter` and `@Setter` for specified field
For example: 
```
@Getter
@Setter
private Long id;
```

Comment (andriimanzar) [reply src/main/java/com/intellias/intellistart/interviewplanning/security/JwtUserDetails.java]: Done!

**Comment by knurff** [comment]
|File|Coverage [65.94%]|:green_apple:|
|:-|:-:|:-:|
|[JwtUtils.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FJwtUtils.java)|100%|:green_apple:|
|[RoleParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Futil%2FRoleParser.java)|100%|:green_apple:|
|[JwtUserDetailsService.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fservice%2FJwtUserDetailsService.java)|100%|:green_apple:|
|[JwtGenerator.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtGenerator.java)|100%|:green_apple:|
|[JwtTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtTokenParser.java)|100%|:green_apple:|
|[FacebookTokenParser.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FFacebookTokenParser.java)|89.29%|:green_apple:|
|[JwtUserDetails.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fsecurity%2FJwtUserDetails.java)|78%|:green_apple:|
|[FacebookTokenDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookTokenDto.java)|0%|:x:|
|[FacebookResponseDto.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fdto%2FFacebookResponseDto.java)|0%|:x:|
|[AuthenticationResponse.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2Fresponse%2FAuthenticationResponse.java)|0%|:x:|
|[AuthenticationController.java](https://github.com/knurff/intellistart-java-2022-NPE_HUNTERS/blob/b2e091dae20b979cd979ab78ec8ed01100ae5ca0/src%2Fmain%2Fjava%2Fcom%2Fintellias%2Fintellistart%2Finterviewplanning%2Fcontroller%2FAuthenticationController.java)|0%|:x:|

|Total Project Coverage|39.62%|:green_apple:|
|:-|:-:|:-:|

**Review by ram333n — APPROVED** [review]

**Pull request merged**